cf: 155494.663
ct: 155494.663
eps: 0.01
mass_fl: 550
mass_fr: 550
mass_rl: 550
mass_rr: 550
max_iteration: 800
ts: 0.02
weight_of_lat_error: 1.0
weight_of_lat_error_rate: 0.05
weight_of_heading_error: 3.0
weight_of_heading_error_rate: 0.1
weight_of_control: 20.0
reverse_weight_of_lat_error: 1.0
reverse_weight_of_lat_error_rate: 0.0
reverse_weight_of_heading_error: 3.0
reverse_weight_of_heading_error_rate: 0.0
look_forward_dis: 0.0
look_forward_time: 0.15
reverse_look_forward_dis: 4.0
filter_param_: 1.0
max_steering_degree: 460.0
wheelbase_: 2.65
wheel_zero_: 0.0
max_delta_steering_degree: 10.0
k_wheel2steering: 16.5
k_minimum_curvature_: 0.02
enable_gain_scheduler: true
